src/platform/site-wide/drupal-static-data/source-files/va-health-services/connect/index.js

```diff
@@ -0,0 +1,10 @@
+import { connectDrupalStaticDataFile } from '../../../connect';
+import { preProcessHealthServicesData } from './preProcess';
+
+export const connectDrupalStaticDataFileVamcEhr = dispatch => {
```

Comment:
I believe we may want this const name adjusted to something like 
```suggestion
export const connectDrupalStaticDataFileVaHealthServices = dispatch => {
```

Reply:
Oops. Absolutely we do.

Reply:
Renamed, thanks!